### PR TITLE
integrations: Add screenshots and notes for Raygun.

### PR DESCRIPTION
**Pull request by Privisus**
[Link to GCI task](https://codein.withgoogle.com/dashboard/task-instances/5568052408090624/).

### CODE REVIEW

File: webhook-integrations/raygun/notes.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+![](screenshots/3.2-reactivated.png)
+
+Repeated error assigned:
+
+![](screenshots/3.3-assigned.png)
+
+Repeated error ignored:
+
+![](screenshots/3.4-ignored.png)
+
```
Comment (eeshangarg) [inline webhook-integrations/raygun/notes.md]: I only have one minor question. The possible messages you list here are a lot more than the number of payloads you submitted as part of your other PR. Do the payloads in that PR cover the possible messages listed here? :)

Comment (Privisus) [reply webhook-integrations/raygun/notes.md]: Yes. Take a look at this example:

```{
  "event":"error_activity",
  "eventType":"StatusChanged",
  "error": {
    "url":"https://app.raygun.com/error-url",
    "description":"Emma marked error Script Error as Ignored",
    "activityDate":"1970-01-28T01:49:36Z",
    "message": "Script Error",
    "status":"Ignored",
    "user":"Emma Cat"
  },
  "application": {
    "name":"Best App",
    "url":"http://app.raygun.io/application-url"
  }
}
```

Notice the key `status` here can be changed into `ignored`, `resolved`, etc. So you really only need one fixture to test it.

Comment (eeshangarg) [reply webhook-integrations/raygun/notes.md]: Ah okay makes sense! Thanks, @Privisus! :)

**Comment by eeshangarg** [comment]
@Privisus: Thanks for working on this! Happy to merge and approve! :)

**Pull request merged**